### ISSUE TITLE
Tiny Tiny RSS integration error

### ISSUE THREAD

**Issue by gitkrax** [lwindolf/liferea#252]
I've tried to use Liferea with Tiny Tiny RSS, but it seems to have a problem with detecting how the feeds are updated.

If I don't press "Update All" none of the feeds are updated. If I press the button, I get the following error: 
```
This TinyTinyRSS source is not self-updating and Liferea does not support updating TinyTinyRSS remotely at the moment. Please change the TinyTinyRSS preferences to automatic updating!
```
This isn't true, since that source is self-updating. I use the `update.php --daemon` command, as [instructed by the install guide](https://tt-rss.org/gitlab/fox/tt-rss/wikis/UpdatingFeeds).

Oddly enough, after I click OK on that dialog, Liferea suddenly notices that Tiny Tiny RSS should be updated and fetches updated feeds.

Versions
* `Liferea 1.10.3`
* `TTRSS v1.15.3`

**Comment by lwindolf** [comment]
This issue was fixed in Liferea 1.10.4. Please always try the latest version before reporting bugs :-)

**Comment by gitkrax** [comment]
I tried going through the changelog, but could not find any mention of this. Thanks!
